.github/workflows/swift-toolchain.yml

```diff
@@ -2715,3 +2728,15 @@ jobs:
           gh release upload ${{ needs.context.outputs.swift_tag }} installer-arm64.exe -R ${{ github.repository }}
           shasum -a 256 installer-arm64.exe > installer-arm64.exe.sha256
           gh release upload ${{ needs.context.outputs.swift_tag }} installer-arm64.exe.sha256 -R ${{ github.repository }}
+
+  upload_size_metrics:
```

Comment:
Rather than doing this, can we not setup a completion trigger on the other job? That would avoid the need to teach this job about the analysis step.

Reply:
If we use workflow_run it adds a good amount of complexity to the the binary size workflow: It needs to run iff this workflow completes with success or if the trigger is not workflow_run, and if the trigger is workflow_run it needs to look up the most recent release from GitHub. Seems far easier to just call it here.

Reply:
This should work fine though: https://github.com/thebrowsercompany/swift-build/pull/148